xds/src/test/java/io/grpc/xds/WeightedRoundRobinLoadBalancerTest.java

```diff
@@ -325,6 +325,15 @@ private void pickByWeight(MetricReport r1, MetricReport r2, MetricReport r3,
         .isAtMost(0.0002);
   }
 
+  private static <T> void incrementCountInMap(Map<T, Integer> pickCount,
```

Comment:
Change in `MultiChildLoadBalancer` looks fine. But the wrr should not be affected.